src/bin/rbw-agent/actions.rs

```diff
@@ -205,6 +205,8 @@ pub async fn login(
         }
     }
 
+    let _ = subscribe_to_notifications(state.clone()).await;
```

Comment:
let's at least write something to the log if this fails

Reply:
Fixed in [d6bbc46](https://github.com/doy/rbw/pull/115/commits/d6bbc46089b519eb82ae009bd48fa6d449f07941)